tests/page-objects/reports/reportsPage.wdio.js

```diff
@@ -0,0 +1,13 @@
+const reportListID = '#reports-list';
+const caseId = () => $(`li.indent-0 p span a`);
+const caseIdLabel = () => $('li.indent-0 label span');
+
+module.exports = {
+  firstReport: () => $(`${reportListID} li:first-child`),
+  submitterName: async () => await (await $('.sender .name')).getText(),
```

Comment:
I see a bit of mix here. Defining `const caseIdLabel = () => $('li.indent-0 label span');` above and then calling`getText()` in the export 

and 
defining the element `submitterPhone: async () => await (await $('.sender .phone')).getText(),` and calling `getText()` in the export. 

Personally I think we should export the element and then interact(click, addValue, getText,etc..) in the spec. It's a little confusing with the mix.

Reply:
done